lib/probe/http-client.js

```diff
@@ -244,6 +244,14 @@ class HttpRequestMetrics {
         if (this.responseSize !== undefined) {
             this.responseSize = parseInt(this.responseSize, 10);
         }
+        // ...otherwise count actual response size as its streamed
+        if (isNaN(this.responseSize)) {
+            this.responseSize = 0;
+            response.on("data", (chunk) => {
```

Comment:
as a temp work around for y'all's GA, could we just add a try/catch around here to avoid throwing if the cosmso client runs into issues with this?

Reply:
Thanks, but the previous release `v1.0.5` is more than enough for us now. I already upgraded the dependency and pinned to `v1.0.5`